### PR TITLE
Fix some bugs and make some improvements for logging in

### PR DESCRIPTION
**Pull request by TimDiekmann**
Please see the commits for each change, every commit only makes a minimal change.

### CODE REVIEW

File: app/static/js/connection.js [hunk 1]
```diff
@@ -67,16 +67,19 @@ function updateUsers() {
...
 
-    socket.on("pong", (data) => {
-        $("#ping").text(data);
-    });
+    setInterval(() => {
+        const start = Date.now();
+        socket.volatile.emit("ping", () => {
+            $("#ping").text(Date.now() - start);
+        });
+    }, 5000);
```
Comment (janagoetze) [inline app/static/js/connection.js:81]: 🏓

**Review by janagoetze — APPROVED** [review]

File: app/static/js/connection.js (hunk 1 above)
Comment (TimDiekmann) [reply app/static/js/connection.js:81]: What's that? :smile:

Comment (janagoetze) [reply app/static/js/connection.js:81]: `:ping_pong:`

Comment (TimDiekmann) [reply app/static/js/connection.js:81]: Oh, my phone could not recognize that emoji 😄

**Pull request merged**